src/devtools/plugin.ts

```diff
@@ -249,7 +249,11 @@ export function registerTresDevtools(app: DevtoolsApp, tres: TresContext) {
                 key: 'matrixWorld',
                 value: instance.matrixWorld,
               },
-
+              {
+                key: 'userData',
```

Comment:
Maybe we can enhance this further and loop on the key-value pairs of the instance instead of going one by one like this:

```ts
payload.state = {
  object: Object.entries(instance).map(([key, value]) => {
    return {
      key,
      value,
      editable: true,
    }
  }).filter(({ key }) => key !== 'parent'),
}
```

Reply:
@alvarosabu absolutely, thanks for the feedback, I'll get on it.

Reply:
Wow I didn't even have to do anything :)

<img width="743" alt="Screenshot 2024-07-07 at 22 52 35" src="https://github.com/Tresjs/tres/assets/6420600/6664ff9d-5e5a-4e7f-bdd6-f9383f9d68f2">